sotopia/cli/benchmark/benchmark.py

```diff
@@ -240,7 +308,7 @@ def run_async_benchmark_in_batch(
                     model_names["test_model"],
                 )
                 rewards_dict["model_name"] = model_names["test_model"]  # type: ignore
```

Comment:
are these two necessary?

Reply:
I think yes? Since the returned `rewards_dict` is `dict[str, tuple[float, float]]`, and the added data are `str` and `int`

Reply:
So this actually violates the type annotation? Can this be avoided?

Reply:
One possible solution is to add int / str type in annotation, but will that make the code unclear?